machine-learning/00_series-intro_outline.md

```diff
@@ -0,0 +1,125 @@
+# Machine Learning
+
+(!) = Definitely needs to be reworded or explained more
+
+Standardized CodeRabbit introduction
+
+Introduction
+------------------------------------------------------------------------------
+* Data is becoming more easily available
+* Data sets are becoming larger
+* Both the private companies and government institutions are becoming better at
+  generating, streaming, and storing data
+ 
+* We need algorithms/models (!) that can help us gain insight from increasingly 
+  large and complex datasets
+
+* Machine Learning is a discipline that studies how computers can automatically 
+  analyze and draw conclusions from data. (!)
+  * Even though technically it is a subfield of mathematics and does not 
+    necessarily need to use computers.
+
+* Examples (I'll probably choose three):
+  * Facebook and Snapchat use machine learning to build technology that can 
+    recognize faces
+  * Online advertisers use machine learning to offer you the products you are 
+    most likely to buy
+  * Large banks use machine learning to detect credit fraud
+  * Self-Driving cars use machine learning to saftely navagate through roads.
+  * Hedge Funds use machine learning to predict movements in the stock market.
+  * Netflix uses machine learning to recommend movies and TV shows you might 
+    like.
+
+Body
```

Comment:
This section can be split out into a "Course Overview" video? 
Each point here can possible become a group of videos (chapter), and each subpoint below can be an individual video. What do you think?